NON_CONTAINERIZED_DEV_README.md

```diff
@@ -1,49 +1,96 @@
-# Developing on SCP without a Docker container
+# Developing on SCP natively, without a Docker container
 
-Developing on SCP without a Docker container, while less robust, opens up some faster development paradigms, including 
-live css/js reloading, faster build times, and byebug debugging in rails.
+Developing on SCP natively, i.e. outside a Docker container, is less robust but faster.  In native mode, full pages load are 4-7x faster, tests run faster, and rich [byebug debugging](https://github.com/deivid-rodriguez/byebug/blob/master/GUIDE.md) in Rails is possible.
 
 ## SETUP
+Commands below assume your CPU is Apple Silicon / M1, not Intel.
+1. Run `rbenv -v`.  If it is not found, `brew install rbenv`, then `rbenv init` to set up rbenv in your shell, then close terminal and reopen it.
+2. Run `ruby -ropenssl -e 'puts OpenSSL::OPENSSL_VERSION'`.  
+  - If it outputs contains "OpenSSL 1.1.1", go to Step 3.  
+  - Else, if output contains something like "OpenSSL 3.0.7", install the needed version of OpenSSL, and install Ruby so that it compiles with the needed version of OpenSSL:
+     - Run `brew install openssl@1.1`
+     - Run `rbenv uninstall 3.1.3`
+     - Run `LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" CPPFLAGS="-I/opt/homebrew/opt/openssl@1.1/include" CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl@1.1)" RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl@1.1)" rbenv install 3.1.3`
```

Comment:
FWIW I think it's safe to have the values for `LDFLAGS`, `CONFIGURE_OPTS`, and `RUBY_CONFIGURE_OPTS` pre-loaded in your shell.  I've been doing this for years without any issue.  That being said, I am still on an Intel chip.  Curious what your thoughts are about having to declare these during install.

My main reason for wanting them in the shell is that I suspect whenever we have to install newer versions of some gems, it will fail with similar errors on M1 machines unless these configuration options are set.  It specifically will address the `bson_ext` error you outlined below.

This is what I have exported in my shell (paths will be different for other people but you get the idea).
```
export CFLAGS="-Wno-error=implicit-function-declaration"
export MAKE_OPTS=-j1 RUBY_CONFIGURE_OPTS="--with-openssl-dir=/usr/local/opt/openssl@1.1 --with-readline-dir=$(brew --prefix readline) --with-libyaml-dir=$(brew --prefix libyaml)"
```

Reply:
So we could suggest the shell modification here in this README (maybe where we're setting CFLAGS="-Wno-error=implicit-function-declaration")? Do those env changes apply to any other build processes (ie. are there other places in our onboarding docs where it should also be noted? or is there a "higher level" process that this shell mod belongs?)

Reply:
Ah okay - I wasn't aware of the previous instances where `export CFLAGS="-Wno-error=implicit-function-declaration"` had resolved other gem install issues. Thanks for the additional context!